driver/bpf/types.h

```diff
@@ -129,17 +129,34 @@ struct sys_stash_args {
 };
 #endif
 
-#ifdef __aarch64__
+#ifdef CAPTURE_SCHED_PROC_EXEC
+
+#ifndef BPF_SUPPORTS_RAW_TRACEPOINTS
```

Comment:
Can this happen? I mean, being inside the `CAPTURE_SCHED_PROC_EXEC` but hot having the `BPF_SUPPORTS_RAW_TRACEPOINTS` (given the check in quirks.h)

Reply:
yes, it could happen, that's exactly why I didn't check for `CAPTURE_SCHED_PROC_EXEC` in `quirks.h`